### DISCUSSION TITLE
como funciona el PBcopysurvey2.0?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#306] [2. Questions and Answers] [answered]
Hola
He visto que el bot copysurvey fue actualizado a una segunda version y me gustaría saber cómo funciona, ya que quiero agregar usuarios a una tarea mediante accessrole y me esta generando un error cuando lo ejecuto.

**Según los campos que leo en el admin web**
![image](https://user-images.githubusercontent.com/61419471/123972503-e70b7700-d988-11eb-92ca-196efaf9631b.png)

**Por base de datos me arroja esto:**
```
{
                "_id" : ObjectId("60dc7562cde2870008630f19"), 
                "key" : "copy_paciente", 
                "name" : "PBCopySurvey", 
                "version" : "2.0.0", 
                "data" : {
                    "formId" : "60db8e24cde2870008625908", 
                    "formChannel" : "60db834fdfb598685a65d90e", 
                    "targetChannel" : "$OUTPUT#crear_tarea_paciente#task|channel", 
                    "sentBy" : "5e0a1a1345e9602068206fdb"
                }, 
                "next" : {
                    "SUCCESS" : "send_pacientes", 
                    "ERROR" : ""
                }, 
                "customNetworkRequest" : [

                ]
            }
```

**Cuando lo ejecuto me arroja esto**
![image](https://user-images.githubusercontent.com/61419471/123973551-dad3e980-d989-11eb-9a1a-df5a28902899.png)

Gracias =>

**Comment by guillevalin** [comment, marked as answer]
@NicolasPLG Tengo entendido que nuestro equipo subió un hotfix para este error.

Podrías validarlo por favor?

Gracias!

**Reply by NicolasPLG** [reply]
lo revisaré aunque ahora me está arrojando esto
```{
key: "copy_pacientes"
name: "PBCopySurvey"
status: "SUCCESS"
result:
messages: Array[0] []
nextBot: ""
iteration: 6
}```

La rutina del bot es ésta

```
{
                "_id" : ObjectId("60dcd2edcde287000863fdc8"), 
                "key" : "copy_pacientes", 
                "name" : "PBCopySurvey", 
                "version" : "2.0.0", 
                "data" : {
                    "formId" : "60db8e24cde2870008625908", 
                    "formChannel" : "60db834fdfb598685a65d90e", 
                    "targetChannel" : "$OUTPUT#crear_tarea_paciente#task|channel", 
                    "createNewAnswer" : true
                }, 
                "next" : {
                    "SUCCESS" : "", 
                    "ERROR" : ""
                }, 
                "customNetworkRequest" : [

                ]
            }
```

:( @guillevalin

**Reply by guillevalin** [reply]
@NicolasPLG No está el resultado de la ejecución de ese bot, pero podemos agendar unos minutos para revisarlo en conjunto, te parece?

**Reply by NicolasPLG** [reply]
hola @guillevalin dale no hay problema lo agendo entonces.

Cierro aqui
